examples/get-started/pure-js/carto/README.md

```diff
@@ -0,0 +1,8 @@
+Pure js demo for CARTO layers
+
+### Usage
```

Comment:
This demo requires a Mapbox token. Either add instructions for acquiring and providing a token (see README of other mapbox examples), or remove the mapbox dependency (preferred for a "minimal" example)

Reply:
Not really since we're using CARTO's basemaap and they are free

Reply:
Remove the mapbox code in this example then?

Reply:
We still use Mapbox GL but tiles are provided by CARTO, so I think we can use Mapbox GL without a token.

Reply:
Do you prefer to have the example without the basemap to make it simpler? I just did because it looks better with the basemap

Reply:
I see, it's fine then